### DISCUSSION TITLE
Launch Modules using gyarpmanager - Writing the console output to a file in windows

### DISCUSSION THREAD

**Discussion by Sharath401** [robotology/community#14] [Q&A] [answered]
Hi,

I would like to know if for a module that is launched using gyarpmanager, is there a possibility of writing the console output to a file in windows?

Assume the following:

Yarp server is already running
A node is created on a WINDOWS workstation ( yarp  run --server /node1)
Given the availability of a module, say Module1, one could use gyarpmanager from YARP to launch the module. Usually a .xml file is specified which looks like:

---

module>
  <name>Module1</name>  
  <parameters</parameters>
  <node>node1</node>
  <stdio></stdio>
  <tag>Module1</tag>
## </module>

When this Module1 is launched using gyarpmanager, the console output that corresponds to Module1 is printed on the console window where /node1 was created. 

The question is, how could one write this console output to a file? ( I am actually interested in the console output of Module1 being written to a file when launched using gyarpmanager)

Any help or suggestion in this context is much appreciated.

Thank you very much for understanding.

Regards,

Sharath

**Comment by pattacini** [comment, marked as answer]
Lately, @randaz81 has made available the `yarplogger` utility to systematically monitor modules output running on a cluster of machines. This might be of some help for you, even though the direct production of a _file_ - if this is your aim - is not addressed, I think... or not @randaz81?

As side note, consider calling `yarpmanager` instead of `gyarpmanager` from now on (after update).
